### ISSUE TITLE
App Will Open Put Stays Minimized

### ISSUE THREAD

**Issue by rrhutch** [saenzramiro/rambox#935]
### Steps to reproduce

1. Uninstalled
2. Tried Reinstalling with both the 64-bit and 32-bit version

### Actual behavior

App will show open on the taskbar, but will not open. Acting like it is staying minimized.

### ENVIRONMENT

> Rambox: v0.5.9

> OS: Windows 10

> Arch: x64

**Comment by nmat** [comment]
hmmmm not sure how you encounter this. Can you check the settings used in preferences to have some more to go on?

Can you also explain if its the trayicon or the taskbar itself?

**Comment by rrhutch** [comment]
It is the taskbar itself. Can’t get into preferences for the app since it won’t open. Here is a link to a short video I recorded of my desktop showing you what it is doing. I have uninstalled and reinstalled about 4 times now, and same result every time. The only thing that is different is that I use multiple monitors at work and just working off my laptop at home, but not sure why that would make a difference. Here is a link to the recording of my desktop: https://www.dropbox.com/s/mplq0o7s85o31eg/ScreenCaptureProject1.mp4?dl=0.

**Comment by rrhutch** [comment]
Got back to the office today and it was an issue with the multiple monitor. Even though there was no secondary monitor connected, it was looking for it. Problem solved.

From: Ryan Hutchinson
Sent: Sunday, June 11, 2017 6:42 PM
To: saenzramiro/rambox; saenzramiro/rambox
Cc: Author
Subject: RE: [saenzramiro/rambox] App Will Open Put Stays Minimized (#935)

It is the taskbar itself. Can’t get into preferences for the app since it won’t open. Here is a link to a short video I recorded of my desktop showing you what it is doing. I have uninstalled and reinstalled about 4 times now, and same result every time. The only thing that is different is that I use multiple monitors at work and just working off my laptop at home, but not sure why that would make a difference. Here is a link to the recording of my desktop: https://www.dropbox.com/s/mplq0o7s85o31eg/ScreenCaptureProject1.mp4?dl=0.
